tests/director/api/fixtures_queries_test.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func SetAutomaticScenarioAssignmentWithinTenant(t *testing.T, ctx context.Context, tenantID, automaticScenario, selecterKey, selectorValue string) {
+	input := graphql.AutomaticScenarioAssignmentSetInput{
+		ScenarioName: automaticScenario,
+		Selector: &graphql.LabelSelectorInput{
+			Key:   selecterKey,
+			Value: selectorValue,
+		},
+	}
```

Comment:
Hope it won't break go fmt
```suggestion
func SetAutomaticScenarioAssignmentWithinTenant(t *testing.T, ctx context.Context, tenantID, automaticScenario, selectorKey, selectorValue string) {
	input := graphql.AutomaticScenarioAssignmentSetInput{
		ScenarioName: automaticScenario,
		Selector: &graphql.LabelSelectorInput{
			Key:   selectorKey,
			Value: selectorValue,
		},
	}
```